kopf/reactor/loading.py

```diff
@@ -14,17 +14,20 @@
 Multiple files/modules can be specified. They will be loaded in the order.
 """
 
+import sys
 import importlib
 import importlib.util
 import os.path
 
 
+
```

Comment:
Python's conventions says it should be 2 empty lines before top-level functions. Can you please fix it here?